OloEngine/src/OloEngine/Core/Log.cpp

```
@@ -5,61 +5,78 @@
 #include <spdlog/sinks/basic_file_sink.h>
 #include <spdlog/sinks/ringbuffer_sink.h>
 
+#if OLO_ASSERT_MESSAGE_BOX
+#ifdef OLO_PLATFORM_WINDOWS
+#include <Windows.h>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)